src/BuiltInTools/dotnet-watch/CommandLine/CommandLineOptions.cs

```diff
@@ -69,12 +75,13 @@ internal sealed class CommandLineOptions
             noHotReloadOption,
             NonInteractiveOption
         ];
-
-        // Options we need to know about that are passed through to the subcommand:
-        var shortProjectOption = new Option<string>("-p") { Hidden = true, Arity = ArgumentArity.ZeroOrOne, AllowMultipleArgumentsPerToken = false };
-        var longProjectOption = new Option<string>("--project") { Hidden = true, Arity = ArgumentArity.ZeroOrOne, AllowMultipleArgumentsPerToken = false };
-        var launchProfileOption = new Option<string>("--launch-profile", "-lp") { Hidden = true, Arity = ArgumentArity.ZeroOrOne, AllowMultipleArgumentsPerToken = false };
-        var noLaunchProfileOption = new Option<bool>("--no-launch-profile") { Hidden = true, Arity = ArgumentArity.Zero };
+        Option[] forwardToRunOptions =
```

Comment:
I'd call this `forwardToSubcommandOptions`

Reply:
You can replace https://github.com/dotnet/sdk/pull/50558/files#diff-6a6fa2fea5af87fda145d79165f529cef772a3227d58494098b7a22511c4aecaR96-R99 with 

```C#
foreach (var option in forwardToSubcommandOptions)
{
     rootCommand.Options.Add(option);
}
```